src/api/operator/numpy/np_matrix_op.cc

```diff
@@ -142,4 +142,158 @@ MXNET_REGISTER_API("_npi.rot90")
   *ret = ndoutputs[0];
 });
 
+MXNET_REGISTER_API("_npi.array_split")
+.set_body([](runtime::MXNetArgs args, runtime::MXNetRetValue* ret) {
+  using namespace runtime;
+  static const nnvm::Op* op = Op::Get("_npi_array_split");
+  nnvm::NodeAttrs attrs;
+  op::SplitParam param;
+  param.axis = args[2].operator int();
+  param.squeeze_axis = false;
+  if (args[1].type_code() == kDLInt) {
+    param.indices = TShape(0, 0);
+    param.sections = args[1].operator int();
+    CHECK_GT(param.sections, 0)
```

Comment:
Do we need to check `shape[axis] % param.sections == 0`? like  https://github.com/apache/incubator-mxnet/pull/17873/files#diff-21ec839e5c3e65fe0e31e75da91d4996R196

Reply:
**_array_split_** support such split which is different from **_split_**